package.json

```diff
@@ -13,7 +13,8 @@
 		"firefox": "node -e \"const fs = require('fs'); fs.copyFileSync('app/manifestV2.json', 'app/manifest.json');\"",
 		"chrome": "node -e \"const fs = require('fs'); fs.copyFileSync('app/manifestV3.json', 'app/manifest.json');\"",
 		"prepare-inpage": "tsc --project tsconfig-inpage-create-injection-script.json && node --enable-source-maps app/inpage/js/create_injection_script.mjs && node --input-type=module -e \"import { promises as fs } from 'fs'; await fs.rm('app/inpage/js/create_injection_script.mjs')\"",
-		"inpage": "tsc --project tsconfig-inpage.json && npm run prepare-inpage && tsc --project tsconfig-inpage-output.json"
+		"inpage": "tsc --project tsconfig-inpage.json && npm run prepare-inpage && tsc --project tsconfig-inpage-output.json",
+		"cleanup": "npx knip"
```

Comment:
could be called something else. I guess this just checks stuff but does not actually do any cleaning?

Reply:
should we do `cleanup:dryrun` and `cleanup` to straight up apply fixes?